twatbot.py

```diff
@@ -493,7 +493,7 @@ def fetch_more_tweets_callback(user, twats):
 
 def scrape(user, first_run = False):
 
-	if first_run and not args.count == -2:
+	if first_run and not os.path.isfile('users/%s/twats.json' % (user.lower())):
```

Comment:
don't we have a function like get_twats_filename ?

Reply:
i think this should be if first_run and (args.count != -2 or not fileexists(...))